runtime/doc/eval.txt

```diff
@@ -6299,15 +6301,19 @@ tempname()					*tempname()* *temp-file-name*
 		For MS-Windows forward slashes are used when the 'shellslash'
 		option is set or when 'shellcmdflag' starts with '-'.
 
-termopen({command}[, {opts}])			{Nvim} *termopen()*
-		Spawns {command} using the shell in a new pseudo-terminal
-		session connected to the current buffer.  This function fails
-		if the current buffer is modified (all buffer contents are
-		destroyed).  The {opts} dict is similar to the one passed to
-		|jobstart()|, but the `pty`, `width`, `height`, and `TERM` fields are
-		ignored: `height`/`width` are taken from the current window and
-		$TERM is set to "xterm-256color".  Returns the same values as
-		|jobstart()|.
+termopen({cmd}[, {opts}])			{Nvim} *termopen()*
+		Spawns {cmd} in a new pseudo-terminal session connected
+		to the current buffer.  {cmd} is the same as the one passed to
+		|jobstart()|.  This function fails if the current buffer is
+		modified (all buffer contents are destroyed).
+
+		The {opts} dict is similar to the one passed to |jobstart()|,
+		but the `pty`, `width`, `height`, and `TERM` fields are
+		ignored: `height`/`width` are taken from the current window
```

Comment:
I can't think of a use case for setting a custom `'height'`, but perhaps `'width'` could be set to the maximum of the buffer's width and the given number so that long commands/output get linewrapped. I seem to remember someone opening an issue about this, but I can't find it.

I'd like to save that for another PR, though. I haven't modified that code.